src/components/SelectedActivities.tsx

```diff
@@ -47,16 +47,29 @@ export function SelectedActivities(props: {
   selectedActivities: Array<Activity>;
   units: number;
   hours: number;
+  hoursFirstHalf: number;
+  hoursSecondHalf: number;
   warnings: Array<string>;
   state: State;
 }) {
-  const { selectedActivities, units, hours, warnings, state } = props;
+  const { selectedActivities, units, hours, hoursFirstHalf, hoursSecondHalf, warnings, state } = props;
+
+  const formatHours = () => {
+    if (hoursFirstHalf === hoursSecondHalf) {
+      return `${hours.toFixed(1)} hours`;
+    }
+    
+    // Use Q1/Q2 for fall semester, Q3/Q4 for spring semester
+    const [firstLabel, secondLabel] = state.term.semester === 'f' ? ['Q1', 'Q2'] : ['Q3', 'Q4'];
+    
+    return `\u2264 ${hours.toFixed(1)} hours (${hoursFirstHalf.toFixed(1)} ${firstLabel}, ${hoursSecondHalf.toFixed(1)} ${secondLabel})`;
```

Comment:
we might as well use ≤ directly in the string

also, what's the rationale for having ≤ here?

Reply:
oh i see the rationale for ≤

can we have a tooltip instead though?